crates/proof_system/global_merkle_root/storage/src/structured_storage/blobs.rs

```diff
@@ -11,3 +18,24 @@ impl MerkleizedTableColumn for BlobData {
         TableColumn::Blobs
     }
 }
+
+impl TableWithBlueprint for BlobData {
+    type Blueprint = Plain<Raw, Raw>;
+    type Column = MerkleizedColumn<TableColumn>;
+
+    fn column() -> Self::Column {
+        MerkleizedColumn::TableColumn(Self::table_column())
```

Comment:
```suggestion
        Self::TableColumn(Self::table_column())
```

Reply:
Unfortunately, this doesn't compile for several reasons:

1.`Self::TableColumn` has to be disambiguated.
2. Disambiguing it doesn't work seems like it seems we can't use a type alias as a constructor.
3. `TableColumn` doesn't implement `StorageColumn`. We'd like to do `Self::Column` instead of `Self::TableColumn` here, but this also fails due to 1. and 2.

Happy to hear if you have any other ideas. I can't think of anything right now that isn't going to result in some serious over-engineering.

Reply:
see https://github.com/FuelLabs/fuel-core/pull/2819/files#diff-2d427bf1c889a9415456528d995366208a39e56422888b2a69a253805efb7679R46

idk why this wouldn't work, compiles on my end :)

Reply:
Done in https://github.com/FuelLabs/fuel-core/pull/2805/commits/adb96afbb7014dbadb2a4bf1998fbe6179505dcf